### PR TITLE
Completed crossword component.

### PR DESCRIPTION
**Pull request by KareemEH**

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -10,13 +10,69 @@ import Menu from "./pages/Menu/Menu.js";
...
 import img from "./images/Earth.png"
+import Crossword from './components/Crossword/Crossword';
 
 function App() {
   const messages = [
     "Hello, This is Captain Cybot! Oh no! Evil malware has taken over the four worlds in our galaxy and each world has been infected with all sorts of malware like viruses, worms, trojans and ransomware. Join Captain Cybot on his adventure to defeat malware and free the galaxy once & for all. Are you ready to join Captain Cybot on his adventure?",
     "If you would like to see more awesome stuff, check out the other writeups at codeworkshop.dev!",
     "Remember to wash your hands!"
   ]
+  const myData = {
```
Comment (JustinWhalley-Carleton) [inline src/App.js:36]: Should be pulled from BE

File: src/components/Crossword/Crossword.js [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+import { IconButton } from "@mui/material";
+import HomeIcon from '@mui/icons-material/Home';
+
+function Crossword(props) {
+    const navigate = useNavigate();
+    const myData = props.data;
+
+    /* Unpack props */
+    //const crosswordData = props.crosswordData;
+    const planet = 'EARTH';
```
Comment (JustinWhalley-Carleton) [inline src/components/Crossword/Crossword.js]: Change to props.planet

File: src/components/Crossword/Crossword.js [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+            <IconButton onClick={() => {navigate("/")}}>
+                    <HomeIcon style={{color: 'white', fontSize: '3.459vw', paddingLeft: '1%', paddingTop: '0.5%'}} />
+            </IconButton>
+            <div>
+                <QuestionContainer>
+                    <QuestionTypeText>Question {questionNumber}: Crossword</QuestionTypeText>
+                    <HowToPlayText>Fill the crossword using the hints.</HowToPlayText>
+                    <Line width={"100%"}/>
+                    <CrosswordContainer>
+                        <ReactCrossword onCrosswordCorrect={crosswordCorrectHandler} data={myData} theme={{gridBackground:'rgb(0,0,0,0)', numberColor:'rgba(0,0,0,0.7)', focusBackground:'#c548ff', highlightBackground:'#d77aff'}}/>
```
Comment (JustinWhalley-Carleton) [inline src/components/Crossword/Crossword.js:58]: Can the css be done in element?

File: src/components/Crossword/CrosswordElements.js [hunk 4]
```diff
@@ -0,0 +1,115 @@
...
+    font-size: 1.5vw;
+`
+
+export const HowToPlayText = styled.h1`
+    padding-bottom: 3.5%;
+    color: #fff;
+    font-weight: 400;
+`
+
+export const ScrmabledWordText = styled.h1`
```
Comment (JustinWhalley-Carleton) [inline src/components/Crossword/CrosswordElements.js]: Unused

File: src/components/Crossword/CrosswordElements.js [hunk 5]
```diff
@@ -0,0 +1,115 @@
...
+    padding-bottom: 10%;
+    padding-top: 10%;
+    color: #fff;
+    font-weight: 800;
+    font-size: 1.953vw;
+`
+
+export const InputDiv = styled.div`
```
Comment (JustinWhalley-Carleton) [inline src/components/Crossword/CrosswordElements.js]: Unused

File: src/components/Crossword/CrosswordElements.js [hunk 6]
```diff
@@ -0,0 +1,115 @@
...
+`
+
+export const BackgroundImg = styled.div`
+    width: 100%;
+    height: 100vh;
+    background-image: url(${props => props.img});
+    background-size: cover;
+`
+
+export const Error = styled.p`
```
Comment (JustinWhalley-Carleton) [inline src/components/Crossword/CrosswordElements.js]: Unused

File: src/components/Crossword/CrosswordElements.js [hunk 7]
```diff
@@ -0,0 +1,115 @@
...
+    color: red;
+    font-size: 0.469vw;
+`
+
+export const ErrorContainer = styled.div`
```
Comment (JustinWhalley-Carleton) [inline src/components/Crossword/CrosswordElements.js]: Unused

File: src/components/Crossword/CrosswordElements.js [hunk 8]
```diff
@@ -0,0 +1,115 @@
...
+    padding-top: 3.5%;
+    height: 100%;
+    width: 100%;
+    text-align: center;
+    padding-bottom: 3.5%;
+`
+
+export const Wrapper = styled.div`
```
Comment (JustinWhalley-Carleton) [inline src/components/Crossword/CrosswordElements.js]: Unused

**Review by JustinWhalley-Carleton — COMMENTED** [review]
I think the PR should be changed to merge into my branch to use the question page

File: src/App.js (hunk 1 above)
Comment (KareemEH) [reply src/App.js:36]: This is here temporarily for the sake of screenshots for the progress report. We will change it to communicate with the BE when it is ready for it.

File: src/components/Crossword/Crossword.js (hunk 3 above)
Comment (KareemEH) [reply src/components/Crossword/Crossword.js:58]: This is the ReactCrossword we got form the dependency it cant be done in element.

**Review by aelsammak — APPROVED** [review]

**Pull request merged**